public/index.html

```diff
@@ -68,6 +68,8 @@
           baseHost: 'http://localhost:3000',
           basePath: '/about-cancer/treatment/clinical-trials/search',
           canonicalHost: 'https://www.cancer.gov',
+					ctsApiEndpointV1: 'https://clinicaltrialsapi.cancer.gov/api/v1',
```

Comment:
this should not have an _/api_, e.g.
ctsApiEndpointV1: 'https://clinicaltrialsapi.cancer.gov/v1',

Reply:
this will probably resolve failing tests

Reply:
Also you need to add test overrides of `ctsApiEndpointV1` into cypress/common/beforeEach, otherwise tests will not use mocks
I think it should be 
ctsApiEndpointV1: '/v1'

Reply:
Yup, missed that out. Thanks!

Reply:
> this should not have an _/api_, e.g. ctsApiEndpointV1: 'https://clinicaltrialsapi.cancer.gov/v1',

Ah, got that wrong. Will fix, thanks.

Reply:
> Also you need to add test overrides of `ctsApiEndpointV1` into cypress/common/beforeEach, otherwise tests will not use mocks I think it should be ctsApiEndpointV1: '/v1'

Yup, missed that. Thanks!